src/dune_rules/merlin/ocaml_index.ml

```diff
@@ -0,0 +1,119 @@
+open Import
+module CC = Compilation_context
+module SC = Super_context
+
+let ocaml_index sctx ~dir =
+  Super_context.resolve_program_memo ~loc:None ~dir sctx "ocaml-index"
+;;
+
+let index_path_in_obj_dir ?for_cmt obj_dir =
```

Comment:
Make this argument required please.

Reply:
Right, it is not even used anymore, I removed it.